src/components/Book.js

```diff
@@ -1,15 +1,35 @@
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/bookSlice';
+
 const Book = (book) => {
+  const dispatch = useDispatch();
   const {
-    type, title, author, chapter,
+    type, title, author, chapter, id,
   } = book;
+  const handledelete = (id) => {
+    dispatch(removeBook({ id }));
+  };
   return (
     <div className="theBook">
       <h3 className="bookType">{type}</h3>
       <h1 className="bookTitle">{title}</h1>
       <h2 className="bookAuthor">{author}</h2>
       <ul className="bookOptions">
         <li className="option">Comments</li>
-        <li className="option">Remove</li>
+        <li>
+          <button
+            className="option"
+            onClick={() => {
+              handledelete(id);
+            }}
+            onKeyDown={() => {
+              handledelete(id);
+            }}
+            type="button"
+          >
+            Remove
+          </button>
+        </li>
         <li className="option">Edit</li>
       </ul>
       <div className="progress">
```

Comment:
- [ ] Good job so far @francis3400, But as you can see below your remove functionality is not working. Once I remove a single book all books are removed and the functionality breaks. Please make sure only a single book is removed when a user click on a removed book.

https://user-images.githubusercontent.com/69077061/224255847-4b3a6789-15ce-42ed-a2b9-491e603dbc0f.mp4